sorc/link_workflow.sh

```diff
@@ -78,7 +78,7 @@ case "${machine}" in
   "gaea")     FIX_DIR="/gpfs/f5/epic/proj-shared/global/glopara/data/fix" ;;
   *)
     echo "FATAL: Unknown target machine ${machine}, couldn't set FIX_DIR"
-    exit 1
+    #exit 0
```

Comment:
Reminder to revert this

Reply:
I kept telling myself to not commit ... but here we are ... good catch again.